.github/workflows/nightly.yml

```diff
@@ -68,13 +68,13 @@ jobs:
     #------------------------------------------------
     - name: Install dependencies
       if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
-      run: poetry install --no-interaction --extras "docs optional_io_formats optional_plotting tests tutorials" --no-root
+      run: poetry install --no-interaction --with dev,docs,optional_io_formats,optional_plotting,tutorials,wbdata --no-root
 
     #------------------------
     #  install root project
     #------------------------
     - name: Install library
-      run: poetry install --no-interaction --extras "optional_io_formats optional_plotting tests tutorials" --only-root
+      run: poetry install --no-interaction --with dev,optional_io_formats,optional_plotting,tutorials,wbdata
```

Comment:
Is there any specific reason to remove `--only-root`? 
The idea is that the step `Install dependencies` just before this one installs the dependencies if they have not been recovered from a cache. Thus, the only thing left to install in the `Install library` step is the project itself. When optional dependencies were defined as extras, they needed to be mentioned here because poetry removes dependencies from unmentioned extras upon `poetry install`. This is not true for groups, though, so we should be fine without `--with ...`. So this lines should be either
`run: poetry install --no-interaction`
or:
`run: poetry install --no-interaction --only-root`, I think.

Reply:
Done